FirebaseCrashlytics.podspec

```diff
@@ -52,23 +55,29 @@ Pod::Spec.new do |s|
     'GCC_C_LANGUAGE_STANDARD' => 'c99',
     'GCC_PREPROCESSOR_DEFINITIONS' =>
       'DISPLAY_VERSION=' + s.version.to_s + ' ' +
-      'CLS_SDK_NAME="Crashlytics iOS SDK" ',
+      'CLS_SDK_NAME="Crashlytics iOS SDK" ' +
+      # For GoogleDataTransport:
+      'PB_FIELD_32BIT=1 PB_NO_PACKED_STRUCTS=1 PB_ENABLE_MALLOC=1 GDTCCTSUPPORT_VERSION=' + s.version.to_s,
```

Comment:
Please remove GDTCCTSUPPORT_VERSION, you're probably redefining it to your own library's version, here and two more times below